var/spack/repos/builtin/packages/py-ics/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyIcs(PythonPackage):
+    """Ics.py : iCalendar for Humans
+
+    Ics.py is a pythonic and easy iCalendar library. Its goals are to
+    read and write ics data in a developer friendly way.
+
+    iCalendar is a widely-used and useful format but not user friendly.
+    Ics.py is there to give you the ability of creating and reading
+    this format without any knowledge of it.
+
+    It should be able to parse every calendar that respects the
+    rfc5545 and maybe some more. It also outputs rfc compliant
+    calendars.
+
+    iCalendar (file extension .ics) is used by Google Calendar, Apple
+    Calendar, Android and many more.
+
+    Ics.py is available for Python>=3.6 and is Apache2 Licensed.
+    """
+
+    homepage = "https://github.com/C4ptainCrunch/ics.py"
+    url      = "https://github.com/C4ptainCrunch/ics.py/archive/v0.5.tar.gz"
```

Comment:
```suggestion
    url      = "https://github.com/C4ptainCrunch/ics.py/archive/v0.6.tar.gz"
```

Reply:
Done, but is this still required? Spack seems to be able to handle anything that follows a typical versioning scheme

Reply:
It's not required for downloading, but the `spack url summary` command will report inaccurate metrics if the URL does not match at least one version in the package.